src/ReactiveUI/ReactiveUI.csproj

```diff
@@ -51,7 +51,11 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'MonoAndroid70' ">
     <Compile Include="Platforms\android\**\*.cs" />
     <Compile Include="Platforms\xamarin-common\**\*.cs" />
-  </ItemGroup>  
+  </ItemGroup>
+  
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
+    <Compile Include="Platforms\netcoreapp1.0\**\*.cs" />
```

Comment:
Related to this, I'd add an additional `PropertyGroup` to `Directory.build.targets` for `netcoreapp`. I think it might make sense to define the `NETFX_CORE` constant there too, but can't remember what exactly that includes off the top of my head. Will double check when I get the chance.

Reply:
That seems fine then, might be we want to make NETFX_CORE two separate defines in future, but don't think it should be in scope for this change.